languages/ruby/exercises/concept/nil/.docs/introduction.md

```diff
@@ -0,0 +1,42 @@
+[Nil][nil-dictionary] is an English word meaning "nothing" or "zero". In Ruby, `nil` is a special value to express the _absence_ of an object.
+
+```ruby
+# I do not have a favorite color
+favorite_color = nil
+```
+
+In other programming languages, `null` or `none` values might play a similar role.
+
+## `nil?`
+
+To determine if a variable holds a `nil` value, the `nil?` method returns a boolean value: `true` is returned if the value is `nil`, `false` otherwise.
+
+```ruby
+favorite_color = nil
+if favorite_color.nil? do
+  "I do not have a favorite color."
+else
+  "My favorite colors is #{favorite_color}."
+end
+# => "I do not have a favorite color."
+```
+
+It is also possible to write an `if` expression on a single line. Note the comma after the condition.
```

Comment:
```suggestion
It is also possible to write an `if` expression on a single line. Note the `then` after the condition.
```

Reply:
"comma" is a relic of the ancestor, I will change, thanks!